store/src/java/com/zimbra/cs/filter/ZimbraAsciiCasemap.java

```diff
@@ -69,8 +69,13 @@ public boolean counts(String operator, List<String> lhs, String rhs)
      */
     public boolean matches(String string, String glob)
             throws SievePatternException {
-        return ZimbraComparatorUtils
-                .matches(string.toUpperCase(), glob.toUpperCase());
+        try {
+            String regex = FilterUtil.sieveToJavaRegex(glob.toUpperCase());
```

Comment:
ZimbraComparatorUtils.matches internally uses FilterUtil.sieveToJavaRegex. thats why used this.